pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -134,7 +135,7 @@ func add(mgr manager.Manager, r *ReconcileNetworkAddonsConfig) error {
 	}
 
 	// Watch for changes to primary resource NetworkAddonsConfig
-	if err := c.Watch(&source.Kind{Type: &opv1alpha1.NetworkAddonsConfig{}}, &handler.EnqueueRequestForObject{}, pred); err != nil {
+	if err := c.Watch(&source.Kind{Type: &cnaov1.NetworkAddonsConfig{}}, &handler.EnqueueRequestForObject{}, pred); err != nil {
```

Comment:
Don't we have to watch for v1alpha1 so in case networkaddonsconfig is allready there we Reconcile it ?

Reply:
good catch!